SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Windows.Shared/Channel/SessionTracker.cs

```diff
@@ -90,24 +89,25 @@ private void HandleEnqueuingLog(object sender, EnqueuingLogEventArgs e)
                     return;
                 }
                 SendStartSessionIfNeeded();
-                e.Log.Sid = _sid == Guid.Empty ? null : new Guid?(_sid);
+                e.Log.Sid = Sid == Guid.Empty ? null : new Guid?(Sid);
                 _lastQueuedLogTime = TimeHelper.CurrentTimeInMilliseconds();
             }
         }
 
         private void SendStartSessionIfNeeded()
         {
             var now = TimeHelper.CurrentTimeInMilliseconds();
-            if (_sid != Guid.Empty && !HasSessionTimedOut(now))
+            if (Sid != Guid.Empty && !HasSessionTimedOut(now))
             {
                 return;
             }
-            _sid = Guid.NewGuid();
+            var previousSid = Sid;
```

Comment:
sorry for the resharping, the actual fix is here and also 2 lines after.

Reply:
Thx for pointing it out. This made the review so much easier! <3